packages/components/src/components/input/input.stories.ts

```diff
@@ -143,6 +143,23 @@ export const Clearable = {
     </div>`
 };
 
+/**
+ * Use the `left` or `right` slot to add system icons.
+ */
+
+export const Icon = {
+  render: () =>
+    html`<div class="flex flex-rows gap-12">
+      <sd-input label="Label" spellcheck>
+        <sd-icon label="landscape" library="global-resources" name="system/picture" slot="left"></sd-icon>
```

Comment:
library="global-resources" is deprecated, no library is needed to be referenced

Reply:
Thanks for the information @auroraVasconcelos. Was this documented somewhere that I may have missed it? I still see lots of places in the codebase with this.

Reply:
@smfonseca, AFAIK this is not in our documentation. Maybe @mariohamann or @Vahid1919 may have more information on this.
I'd like to correct myself; earlier, I said _deprecated_ but actually, _library="global-resources"_ is legacy.